static/app/components/events/eventTags/index.tsx

```diff
@@ -31,18 +32,20 @@ export function EventTags({event, organization, projectId, location}: Props) {
   const streamPath = `/organizations/${orgSlug}/issues/`;
 
   return (
-    <Pills>
-      {event.tags.map((tag, index) => (
-        <EventTagsPill
-          key={!defined(tag.key) ? `tag-pill-${index}` : tag.key}
-          tag={tag}
-          projectId={projectId}
-          organization={organization}
-          query={generateQueryWithTag(location.query, tag)}
-          streamPath={streamPath}
-          meta={meta?.[index]}
-        />
-      ))}
-    </Pills>
+    <ClippedBox clipHeight={150}>
```

Comment:
I feel like we should make this component less gross when it expands

Reply:
its animated, but i have one improvement in mind for when it only hides like 5px and reveals nothing